playground/pages/index.vue

```diff
@@ -0,0 +1,68 @@
+<script setup>
+import { ref } from 'vue'
+const CreditCardError = ref()
+const CreditCardToken = ref()
+
+const IdealError = ref()
+</script>
+<template>
+    <h1>Nuxt mollie payments components</h1>
+
+    <h2>Credit Card</h2>
+
+    <ShopwareFrontendsCreditCard
+        submit-button-label="Save"
+        locale="en_US"
+        :submit-disabled="!!CreditCardToken"
+        @submit="
+            (token) => {
+                CreditCardToken = `${token} ✔️`
+                CreditCardError = null
+            }
+        "
+        @error="
+            (message) => {
+                CreditCardError = `${message} ❌`
+            }
+        "
+    />
+
+    <div class="demo-mollie-results">
+        <div>
+            Test Credit Number:
+            <pre>2223 0000 1047 9399</pre>
+        </div>
+        <hr />
+        <div v-if="CreditCardError">Error: {{ CreditCardError }}</div>
+        <div v-if="CreditCardToken">Token: {{ CreditCardToken }}</div>
+    </div>
+
+    <h2>Ideal</h2>
+
+    <ShopwareFrontendsIdeal
```

Comment:
The linter complains here

`The template root requires exactly one element`

Reply:
This rule only applies to Vue2, in Vue3 this is possible :)
But there is no harm in wrapping it in a single root, I'll do that to make the linter happy :) 

https://eslint.vuejs.org/rules/no-multiple-template-root#rule-details
> This rule checks whether template contains single root element valid for Vue 2.

Reply:
fixed :)

Reply:
Great, thanks! :)